### PR TITLE
Fix stale URL causing build failure

### PR DESCRIPTION
**Pull request by analogue**

### CODE REVIEW

**Review by chriskuehl — APPROVED** [review]

File: requirements-dev.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-coverage
+coverage<5  # Remove constraint once off python 3.6.0 on xenial
```
Comment (asottile) [inline requirements-dev.txt:1]: this isn't correct -- `python_requires` should take care of this for you

Comment (analogue) [reply requirements-dev.txt:1]: Adding `python_requires='!=3.6.0'` to `setup.py` would mean aactivator can't be used with python 3.6.0 which is not the intention. 

coverage is a dev dependency and succumbs to a specific bug in python3.6.0 on xenial only, iirc.

Am I missing something in your comment?

Comment (asottile) [reply requirements-dev.txt:1]: yes, coverage itself [specifies `python_requires`](https://github.com/nedbat/coveragepy/blob/66173dc24db5e6800483e0faddf583e80d9eb9b3/setup.py#L136)

Comment (analogue) [reply requirements-dev.txt:1]: Oh, your proposing a change to coverage. Didn't get that from initial comment.

**Pull request merged**